qa/integration-tests/src/test/java/io/camunda/it/client/UserTaskQueryTest.java

```diff
@@ -30,13 +30,13 @@ class UserTaskQueryTest {
   private static Long userTaskKeyTaskAssigned;
 
   @TestZeebe
-  private static TestStandaloneCamunda testStandaloneCamunda = new TestStandaloneCamunda();
+  private static final TestStandaloneCamunda TEST_STANDALONE_CAMUNDA = new TestStandaloneCamunda();
```

Comment:
@houssain-barouni, I faintly remember you were working on the test setups so they are retriable. Does this here need an adjustment?

Reply:
Hi @tmetzke , that's right, but we need first to merge https://github.com/camunda/camunda/pull/22042

Reply:
Removed the _final_ identifier from the Zeebe setup. Will merge this once CI passes, thanks everyone!